apps/meteor/ee/server/settings/contact-verification.ts

```diff
@@ -0,0 +1,39 @@
+import { settingsRegistry } from '../../../app/settings/server';
+
+export const addSettings = async (): Promise<void> => {
+	const omnichannelEnabledQuery = { _id: 'Livechat_enabled', value: true };
+
+	return settingsRegistry.addGroup('Omnichannel', async function () {
```

Comment:
we hav some settings on livechat-enterprise too, shouldn't these be there too?

Reply:
I moved those here because they should be created even on CE (just disabled)